### PR TITLE
YTDL tag update

### PR DESCRIPTION
**Pull request by kkrypt0nn**
As per rule 5:
> Do not provide or request help **on projects** that may break laws, breach terms of services, or are malicious or inappropriate.

The current tag says you should not assist with *questions related to ytdl*. People might ask a simple question about general Python coding within a project breaking YouTube's Terms of Service, and other members provide support for the user, which breaks the rule 5.

### CODE REVIEW

**Comment by JacobMonck** [comment]
Which issue is this relating to?

**Comment by kkrypt0nn** [comment]
Not really any, the current tag simply doesn't really say the same as the rules.

**Comment by JacobMonck** [comment]
So normally for PR's you need to make an issue so we can discuss and bounce back ideas and agree on what ends up in the repo. I believe we have some contributing guidelines in the readme of this repo.

**Comment by JacobMonck** [comment]
And personally I don't see what the issue is with this but if we were to change it I would do something like "questions/projects" but that's just my opinion.

**Comment by Xithrius** [comment]
Until the issue gets an approval, this PR shall not be merged.

**Review by Kronifer — CHANGES_REQUESTED** [review]
maybe reword this to specify both questions and projects? Otherwise, I think this should be good.

**Comment by onerandomusername** [comment]
> maybe reword this to specify both questions and projects? Otherwise, I think this should be good.

This edit is currently blocked until a dev actually approves it, see the comment by xith

**Comment by JacobMonck** [comment]
> maybe reword this to specify both questions and projects? Otherwise, I think this should be good.

I agree

**Review by Bluenix2 — APPROVED** [review]
